src/guides/v2.3/ui_comp_guide/components/ui-container.md

```diff
@@ -0,0 +1,105 @@
+---
+group: ui-components-guide
+title: Container component
+---
+
+The Container [UI component](https://glossary.magento.com/ui-component) can be used for wrapping particular fields or other ui-components in it with some description text. It can also be used without any field but just to display a notice or description of a fieldset.
+
+## Configuration options
+
+Component's options are set in the UI-component's configuration `.xml` file as follows:
+
+```xml
```

Comment:
Please, move the example to the bottom of the `Configuration options` section so we have the table with options at the very beginning. We follow this concept in other UI-component related articles, so it would be nice to have the same look and feel everywhere. 

Thank you!